mapstruct-2/src/main/java/com/baeldung/mapstruct/iterable/DepartmentMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.mapstruct.iterable;
+
+import java.util.List;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import com.baeldung.mapstruct.iterable.model.Department;
+import com.baeldung.mapstruct.iterable.model.Employee;
+
+@Mapper
+public interface DepartmentMapper {
+    DepartmentMapper INSTANCE = Mappers.getMapper(DepartmentMapper.class);
+
+    // This attempt of mapping causes error, preventing code from compiling
```

Comment:
Can we remove this commented code if not required?

Reply:
Not required, removed.